code/modules/hydroponics/grown.dm

```diff
@@ -200,24 +200,7 @@
 				qdel(src)
 				return
 			else if(seed.chems)
-				if(W.sharp && W.edge && !isnull(seed.chems["woodpulp"]))
-					user.show_message("<span class='notice'>You make planks out of \the [src]!</span>", 1)
```

Comment:
I would leave this block in, as woodpulp can exist in plants that don't specifically have the item-product for logs. It makes sense for Towercaps to go straight to logs, but some other plants might need processing.

Reply:
Alright. Was just thinking it'd be mostly redundant, but will fix.